### ISSUE TITLE
Move or duplicate animation properties from preferences

### ISSUE THREAD

**Issue by papio-pl** [opentoonz/opentoonz#3439]
Hi,

Is it difficult to move these two settings:

![obraz](https://user-images.githubusercontent.com/19463849/89834143-8271de80-db62-11ea-9aa5-15279ca018b6.png)

... to a Tool Options Bar, when animating (moving, rotating, sizing, etc.) or alternatively to some floating new window which you can dock into the workspace? Like that:

![obraz](https://user-images.githubusercontent.com/19463849/89835022-32941700-db64-11ea-8606-d17454ac45d4.png)

I change those settings a lot while animating, and I have to open preferences over and over again:)

Thanks.

**Comment by DarrenTAnims** [comment]
I fully agree. I change them a lot too. Especially the interpolation type. I tend to change the number less. But I'd go further and suggest that these should be shown at the left of the toolbar, possibly just after the first dropdown.

**Comment by artisteacher** [comment]
On small screens, there's already way to many tool options to fit. I'd rather see this added to the command bar.

**Comment by RodneyBaker** [comment]
I'm tagging this with roadmap because @manongjohn has been making progress in this area in T2D.
It may take some time for that effort to mature and be ported but moving in that direction is fairly inevitable.
